R/formatter.r

```diff
@@ -158,6 +162,9 @@ scientific <- function(x, digits = 3, ...) {
 #' ordinal(1:10)
 
 ordinal_format <- function(x) {
```

Comment:
Should this even have an argument?

Reply:
No, good point. It shouldn't. I am going to remove that argument.